hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/ReadableUpgradeStoreImpl.java

```diff
@@ -69,13 +73,13 @@ public String getFileStateKey() {
     @Override
     @Nullable
     public File peek() {
-        return upgradeFileState.get();
+        return upgradeFileState.get(UPGRADE_FILE_ID);
     }
 
     @Override
     @NonNull
     public Iterator<File> iterator() {
-        return List.of(upgradeFileState.get()).iterator();
+        return List.of(upgradeFileState.get(UPGRADE_FILE_ID)).iterator();
```

Comment:
This is my error.  List.of will throw if the input is null, so here's a suggested change.
```suggestion
        final File upgradeFile = upgradeFileState.get(UPGRADE_FILE_ID);
        return upgradeFile != null 
                ? List.of(upgradeFile).iterator()
                : new ArrayList<File>(0).iterator();
```

Reply:
fixed thanks